elvis.config

```diff
@@ -27,7 +27,8 @@
                  %% Allow usage of ?assertMatch and similar macros, since after expanding they
                  %% result in use of variables like __X and __V that wouldn't be permitted by
                  %% default regex.
-                 {elvis_style, variable_naming_convention, #{regex => "^(_?[A-Z][0-9a-zA-Z_]*)$"}}
+                 {elvis_style, variable_naming_convention, #{regex => "^(_?[A-Z][0-9a-zA-Z_]*)$"}},
+                 {elvis_style, macro_names, disable}
```

Comment:
In tests? Yes. There are lots of "standard" test macros that don't make Elivis happy, and there is no way to change regex like for `variable_naming_convention `.